core/pubnub_generate_uuid.c

```diff
@@ -46,7 +46,11 @@ int pubnub_generate_uuid_v1_time(
     ud->clock_seq_low = *io_clock_seq & 0xFF;
     ud->clock_seq_hi_and_reserved = (*io_clock_seq & 0x3F00) >> 8;
     ud->clock_seq_hi_and_reserved |= 0x80;
+#ifdef ESP_PLATFORM
+    memcpy(ud->node, i_node, sizeof ud->node);
```

Comment:
Since there is a conditional macro, it looks like there is some difference in `memcpy` interface for `ESP` and other platforms. Do we pass values, not pointers?

Reply:
Good catch. I don't know why I've made that change. 
I will change back that into previous value.